cmd/glasskube/cmd/list.go

```diff
@@ -133,6 +130,15 @@ func handleEmptyList(resource string) {
 	}
 }
 
+func allPkgs(clpkgs []*list.PackageWithStatus, pkgs []*list.PackagesWithStatus) []*list.PackageWithStatus {
+	result := make([]*list.PackageWithStatus, 0, len(clpkgs)+len(pkgs))
+	result = append(result, clpkgs...)
+	for _, pkg := range pkgs {
+		result = append(result, pkg.Packages...)
```

Comment:
The problem with this approach is, that when a package is not installed at all, it will not appear in the resulting list at all (because `pkg.Packages` is empty). On the other hand, if a certain clusterpackage is not installed, it will be part of the resulting list. So that would be inconsistent. 

I think you would need to check if `pkg.Packages` is empty here. If not, you can do what you are doing right now, but if it is empty, you would need to construct an "empty" `list.PackageWithStatus`, where you just set the `MetaIndexItem`.

```
if len(pkg.Packages) > 0 {
	result = append(result, pkg.Packages...)
} else {
	result = append(result, &list.PackageWithStatus{
		MetaIndexItem: pkg.MetaIndexItem,
	})
}
```